.idea/gradle.xml

```diff
@@ -3,9 +3,8 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="LOCAL" />
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
```

Comment:
I'm using the gradle wrapper and you're using a local gradle installation -- I checked my local version and it's version 2.14, which is behind the minimum supported version (2.14.1). I don't think using the wrapper has any disadvantages though.